static/scripts/loggedin.js

```diff
@@ -1,5 +1,4 @@
 if (window.opener && window.opener !== window) {
-    document.body.classList.add('window-inline');
     window.isInline = true;
```

Comment:
Is this still needed?

Reply:
Yes it ist, because in the content.js the `window.isInline` property is needed for directly adding content-items to the topic editor

https://github.com/schul-cloud/schulcloud-client/blob/master/static/scripts/content.js#L45

And I think we will need it in some other places in future.

Reply:
Ok, than this one is good for merge and rollout imho